### ISSUE TITLE
Broadlink RM PRO PLUS and SP3, SP3S

### ISSUE THREAD

**Issue by BibiMaxx** [RWensveen/com.broadlink#4]
Hi Remco

When I added the rm Pro device for the first time in the app he recognized it now I want to add it again in the new version and i got the message shows" another device: RM2 PRO PLUS (10026)" 
As you know  i tried the sp2 for the sp3 and then it's worked and now in the new version i got the the message " other device found: OEM Branded SPmini (10035)"
Also i tried the SP3S and it's work  but only the night switch is not there

Best regards

**Comment by RWensveen** [comment]
@BibiMaxx 
SP3S: it was not clear to me what the features of the SP3S were, compared to a SP2. I thought it had no nightlight. So I added the nightlight feature to the SP3S.
SP2: small mistake on my side: it tries to verify a device id against the one returned by the detected device. I used the wrong device id in the SP2 driver :-(    I have to use the RM3mini as substitute for most devices, as I do not have any. luckely you do :-)
